resources/rpi/user-data

```diff
@@ -0,0 +1,76 @@
+#cloud-config
+# https://cloudinit.readthedocs.io/
+
+# Set the first part of the name to camera device.
+hostname: huntsmanpi000
+
+# If you have set up ssh on github you can pull down your
+# key automatically so that you can log into the unit without
+# a password. Change 'CHANGEME' to your username below.
+ssh_import_id:
+  - gh:huntsmanscope
```

Comment:
@lspitler do you own the `huntsmanscope` user?  We can do make some things slightly easier with ssh keys if you have a public ssh for that account.  If not we can create one.